src/mame/includes/fmtowns.h

```diff
@@ -138,36 +138,46 @@ class towns_state : public driver_device
 	void towns(machine_config &config);
 	void townsftv(machine_config &config);
 	void townshr(machine_config &config);
+	void townsmx(machine_config &config);
 	void townssj(machine_config &config);
 
+	DECLARE_WRITE_LINE_MEMBER(towns_scsi_irq);
+	DECLARE_WRITE_LINE_MEMBER(towns_scsi_drq);
 	INTERRUPT_GEN_MEMBER(towns_vsync_irq);
-
+	DECLARE_READ16_MEMBER(towns_scsi_dma_r);
+	DECLARE_WRITE16_MEMBER(towns_scsi_dma_w);
+	
```

Comment:
Why have you moved these to the public section?  Aren't they only accessed from machine configuration member functions?

Reply:
I meant to move them to protected after testing some ideas, and use the derived class on the machine configuration, but I never actually did it. Fixed.